### PR TITLE
Add pkg for TO-92 & TO-92-WIDE

### PR DESCRIPTION
**Pull request by ubruhin**
##### SUMMARY

Probably one of the most used package of THT boards but still missing in the library :see_no_evil: 

The TO-92 package is used with legs bend in different ways. Either straight (non-bend), zigzag (TO-18 compatibility according Wikipedia) or bend to a larger pitch. Usually we would create only one package with different footprint variants for differently bend legs. However, this part is actually sold in different variants so bending does not necessarily needs to be done manually. At least non-bend and larger pitch are available, for zigzag I'm not sure.

Therefore, I think it makes sense to threat these as separate packages. For now, I created packages "TO-92" (straight legs) and "TO-92-WIDE". However, for private projects one might still bend the straight legs to get a larger pitch (for easier soldering) so I added footprint variants for all three bend styles to the "TO-92" package.

It's not perfect, but this way at least the default footprint will always correspond to the bend style of the actually sold part. If the user manually switches to a different footprint, it should be clear for him that this requires to bend the legs manually.

**TO-92-WIDE**:
image
image

**TO-92**:
image
image
image
image
image

##### DATASHEETS / REFERENCES

- Wikipedia: https://en.wikipedia.org/wiki/TO-92
- JEDEC TO-092: https://web.archive.org/web/20170618095849/https://www.jedec.org/sites/default/files/docs/archive/to/to-092.pdf

##### OPEN QUESTIONS / UNRESOLVED ISSUES

See thoughts above.

##### CHECKLIST

- [x] I have read and followed the library conventions¹.
- [x] For packages, I followed IPC7351C (see details in library conventions).
- [x] For modified elements, I bumped their version number².
- [ ] I already used the new library elements in a design and verified their correctness (optional).
- [x] I'm the copyright owner of the added content (i.e. the changes are made by myself, not copied/imported from somewhere else).
- [x] I agree to publish all my changes under the CC0 Public Domain License³, allowing everyone to use and modify the content without any restrictions.

¹ *Library Conventions: https://docs.librepcb.org/#libraryconventions*
² *Minor version bump if only metadata was modified (e.g. "0.1" -> "0.1.1"), major version bump if functional changes were made (e.g. "0.1" -> "0.2")*
³ *CC0 Public Domain License: https://en.wikipedia.org/wiki/CC0*

### CODE REVIEW

File: pkg/5ee92b46-ab92-4eda-9785-99ef7a36da9f/package.lp [hunk 1]
```diff
@@ -0,0 +1,357 @@
+(librepcb_package 5ee92b46-ab92-4eda-9785-99ef7a36da9f
+ (name "TO-92")
+ (description "Transistor outline package as specified by JEDEC TO-092 with straight (non-bend legs). For parts sold with bended legs for a larger pitch, use the \"TO-92-WIDE\" package instead!")
```
Comment (dbrgn) [inline pkg/5ee92b46-ab92-4eda-9785-99ef7a36da9f/package.lp:3]:
```suggestion
 (description "Transistor outline package as specified by JEDEC TO-092 with straight (non-bent legs). For parts sold with bent legs for a larger pitch, use the \"TO-92-WIDE\" package instead!")
```

File: pkg/771d5bda-eda6-4eb8-8410-46b117e3c923/package.lp [hunk 2]
```diff
@@ -0,0 +1,159 @@
+(librepcb_package 771d5bda-eda6-4eb8-8410-46b117e3c923
+ (name "TO-92-WIDE")
+ (description "Transistor outline package as specified by JEDEC TO-092, with legs bended for a larger pitch. Only use this package for parts which are sold with bend legs!")
```
Comment (dbrgn) [inline pkg/771d5bda-eda6-4eb8-8410-46b117e3c923/package.lp:3]:
```suggestion
 (description "Transistor outline package as specified by JEDEC TO-092, with legs bent for a larger pitch. Only use this package for parts which are sold with bent legs!")
```

File: pkg/5ee92b46-ab92-4eda-9785-99ef7a36da9f/package.lp [hunk 3]
```diff
@@ -0,0 +1,357 @@
...
+ (keywords "to92,to-092")
```
Comment (dbrgn) [inline pkg/5ee92b46-ab92-4eda-9785-99ef7a36da9f/package.lp:4]: According to Wikipedia, this package is also known as SOT54:

```suggestion
 (keywords "to92,to-092,sot54,sot-54")
```

File: pkg/771d5bda-eda6-4eb8-8410-46b117e3c923/package.lp [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+ (keywords "to92,to-092")
```
Comment (dbrgn) [inline pkg/771d5bda-eda6-4eb8-8410-46b117e3c923/package.lp:4]: According to Wikipedia, this package is also known as SOT54:

```suggestion
 (keywords "to92,to-092,sot54,sot-54")
```

File: pkg/5ee92b46-ab92-4eda-9785-99ef7a36da9f/package.lp [hunk 5]
```diff
@@ -0,0 +1,357 @@
...
+   (auto_rotate true) (mirror false) (value "{{NAME}}")
+  )
+  (stroke_text 849fabb3-2d7f-48d5-a88b-32b87d2d9e64 (layer top_values)
+   (height 1.0) (stroke_width 0.2) (letter_spacing auto) (line_spacing auto)
+   (align center top) (position 0.0 -2.54) (rotation 0.0)
+   (auto_rotate true) (mirror false) (value "{{VALUE}}")
+  )
+ )
+ (footprint a8705313-f284-4dc2-a80b-11dc0ae3ba6f
+  (name "Zigzac (TO-18)")
```
Comment (dbrgn) [inline pkg/5ee92b46-ab92-4eda-9785-99ef7a36da9f/package.lp:79]:
```suggestion
  (name "Zigzag (TO-18)")
```

**Review by dbrgn — CHANGES_REQUESTED** [review]
(Looks good otherwise!)

**Comment by ubruhin** [comment]
Thanks for the fixes! :tada:

**Pull request merged**